runtime/parallel/src/lib.rs

```diff
@@ -397,6 +398,21 @@ impl pallet_liquid_staking::Config for Runtime {
     type BaseXcmWeight = BaseXcmWeight;
 }
 
+parameter_types! {
+    pub StakingPoolAccount: AccountId = StakingPalletId::get().into_account();
+}
+
+pub struct ToStakingPool;
+impl TakeRevenue for ToStakingPool {
```

Comment:
Where does the revenue come from?

Reply:
it comes from xcm dest weight @kaichaosun